eng/pipelines/templates/steps/analyze.yml

```diff
@@ -46,3 +46,9 @@ steps:
       CheckLinkGuidance: $true
 
   - template: /eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml
+
+  - template: /eng/common/pipelines/templates/steps/verify-changelog.yml
```

Comment:
It seems there is exactly one package per service for go. Is that always the case. If not you will have to run this for each artifact

Reply:
In the case of azcore and azidentity I suspect you want either the serviceName or the packageName to be empty given there is only one directory layer.